content/concepts/content-management/04-content-templates.md

```diff
@@ -4,4 +4,48 @@ metaTitle: "sensenet - Content Templates"
+Content templates can reside in the following places:
+
+- global templates: /Root/ContentTemplates
+- local templates: in ContentTemplates system folders on Site, Workspace or local content level
+
+Every content template resides in a folder named by its content type. In this folder there can be multiple content templates - like multiple document templates in the /Root/ContentTemplates/File folder. Templates can have any name but it is advisable to give a name that reflects the type of the template.
+
+# Creating a template
+
+Content templates are content of the same type as they are defined for. For example document templates are files; library or list templates are document libraries or memo lists. To create a content template you simply need to create a new content under the appropriate folder (named by the type). You can set any property of the content template or add child content (e.g. custom views for libraries), they will be part of the template.
+
+# Modify or deleting a template
+
+You can modify or delete a content template at any time. They are not connected to the content created using them in any way. For this reason, modifying a template will not modify a content previously created.
+
+# Creating a new content with a template
+
+Content templates are recognized and cached by the system immediately. You can create new content using the templates in any folder where that particular type is allowed.
+When you open a New menu anywhere it will contain the following items for every allowed type:
+- if the content type has no content template, the type name will be listed (e.g. New Memo)
+- in case there are existing templates for that type all relevant templates will be listed (for example if the File content type is allowed in a document library, all the document templates will appear in the New menu with their name displayed)
+
+After selecting a content template the user will be presented a content view for that content type and will be able to see and change the predefined template fields (depending on the content view). When the user presses the Done editing button the whole template will be copied to the destination place. E.g. if the template is of the workspace type, all its lists and libraries will be present at the destination place.
+
+See example under API docs [https://docs.sensenet.com/api-docs/content-management/01-create]
+
+# Permissions and security
+
+Content templates can be controlled by permissions the same way as any other content. If the user has no permission for a particular template she will not be able to create content using that template.
+After creating a content using a template the security settings that were set directly on the content template are also copied to the new content. All settings set on the destination parent will be inherited by the new content naturally.
+
```

Comment:
local groups can be also added to workspace templates so this way the user is able to build separate micro permission system locally